src/easy/handler.rs

```diff
@@ -2049,7 +2050,14 @@ impl<H> Easy2<H> {
     /// `CURLOPT_CONNECTTIMEOUT_MS`.
     pub fn connect_timeout(&mut self, timeout: Duration) -> Result<(), Error> {
         let ms = timeout.as_secs() * 1000 + timeout.subsec_millis() as u64;
```

Comment:
While not 100% related to this change, while you're here would you be up for using [`Duration::as_millis`](https://doc.rust-lang.org/stable/std/time/struct.Duration.html#method.as_millis) here instead? The fallible u128-to-`c_long` conversion below will automatically handle out-of-range values of `Duration` itself.